recipes/example/meta.yaml

```diff
@@ -1,60 +0,0 @@
-# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
```

Comment:
Interestingly this causes AppVeyor to fail.

While one could look at that as useful information, it feels a bit noisy and indirect. Building on Windows is kind of challenging anyways. So why would one assume the `example` is the cause. 😲

If we really wanted to provide this information, we shouldn't provide it through a CI failure. We should be far more direct and clear about it. For instance, we could have the linter bot verify the `example` is here. Not that I think this is necessary.

Furthermore it is pretty obvious when the example is removed in a PR. It is one entire file that has been deleted. One can easily find this out in the diff as I just did. No need to have an additional less than obvious AppVeyor failure associated.

As a result, we should get rid of this odd AppVeyor errror.